src/aiortc/contrib/media.py

```diff
@@ -360,6 +393,8 @@ class MediaRecorder:
 
     def __init__(self, file, format=None, options={}):
         self.__container = av.open(file=file, format=format, mode="w", options=options)
+        self.received_video_frame = -1
```

Comment:
do you want to start at -1? can we start at 0? feels cleaner

Reply:
removed this variable, it's no longer needed

Reply:
started from 0 for self.received_keypoints_frame